spec/concepts/api/v1/serializer/error_spec.rb

```diff
@@ -8,23 +8,34 @@
     let(:error_object) { instance_double('ErrorleObject', path: [pointer], text: detail) }
     let(:pointer) { :some_pointer }
     let(:detail) { 'some detail' }
+    let(:json_schema) do
```

Comment:
Honestly, it's not a json schema.

Reply:
I agree.I have replaced this with error_hash